src/components/Api.js

```diff
@@ -1,3 +1,145 @@
 export default class Api {
-  
-}
\ No newline at end of file
+  constructor() {
+    this.baseURL = "https://around.nomoreparties.co";
+  }
+
+  getUserInfo () {
+    return fetch(`${this.baseURL}/v1/group-12/users/me`, {
+      headers: {
+        authorization: "397bd50b-9f39-4bee-ad9c-11e69aa20ec4"
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)